book_source/developers_guide/Adding-an-Input-Converter.Rmd

```diff
@@ -1,29 +1,30 @@
-General Note: dates in the database should be datatime (preferably with timezone), and datetime passed around in PEcAn should be of type POSIXlt.
+# How to Add an Input Converter
 
-# Met Data
+## Meterological Data conversion
 
-The standard met data inputs should be of the form:
+In general, you will need to write a function to download the raw met data andone to convert it to the PEcAn standard.
 
-Converters from a raw to standard format go in `/modules/data.atmosphere/R`; converters from standard to model-specific go in `models/<mymodel>/R`.
+Downloading raw data function are named `download.<source>.R`. Example functions can be found within the PEcAn directory [`/modules/data.atmosphere/R`](https://github.com/PecanProject/pecan/tree/develop/modules/data.atmosphere/R). 
 
-* For a number of common gridded products (NARR, CRUNCEP, ISIMIP), there are bash scripts for converting and rechunking in this repository: https://github.com/ebimodeling/model-drivers; these are not generalized, but may be merged into PEcAn at some point.
+Conversion function from raw to standard are named  `met2CF.<source>.R`. Example functions can be found within the PEcAn directory [`/modules/data.atmosphere/R`](https://github.com/PecanProject/pecan/tree/develop/modules/data.atmosphere/R).
 
-Examples:
-* NARR:
-* CRUNCEP:
-* ISIMIP: 
+Current Meteorological products that are coupled to PEcAn can be found in our [Available Meterological Drivers] page.
 
-Names should be `met2CF.<sourcename>` and `met2model.<modelname>`.
+Note: You will not need to write a script to convert from PEcAn standard to PEcAn models. Those conversoin scripts are written when a model is added and can be found within each model's PEcAn directory.
```

Comment:
Typo "conversoin"

Reply:
"Unless you are also adding a new model, you will not need to write..."